src/test/unit/tests/validators/autoCompleteFormValidator.test.ts

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
+import { FormValidator } from '../../../../main/validators';
+import { DataManagerDataObject } from '../../../../main/interfaces';
+import config from 'config';
+
+const dataProcessorResultJson: DataManagerDataObject[] = JSON.parse(
+  fs.readFileSync(__dirname + '/../../data/form-validator-results.json', 'utf-8')
+);
+
+describe('FormValidator', () => {
+  const parent: DataManagerDataObject = dataProcessorResultJson.filter(
+    (item: DataManagerDataObject) => item.id === 'PF0001-RA0001-RA0008-RA0042'
+  )[0];
+
+  const processedItems: DataManagerDataObject[] = dataProcessorResultJson.filter(
+    (item: DataManagerDataObject) => item.id === 'PF0001-RA0001-RA0008-RA0042'
+  );
+
+  describe('Validate', () => {
+    test('should fail radio other validation', async () => {
+      if (
+        !parent._other &&
+        parent._listOfValuesLength &&
+        parent._listOfValuesLength >= JSON.parse(config.get('radio.listOfValuesLength'))
+      ) {
```

Comment:
why do you need this condition here? the same question to all below...

Reply:
It will just to give us clue what we are testing here without looking into form-validator-results.json. And see in code which one it is referring. The condition will always be true as that's what has been put in orm-validator-results.json

Reply:
why not make a better description of test, or put a comment telling what we are testing here? this condition check that is always true doesn't make any sense, at least to me.. code implies that it is needed, however if we take it out, absolutely nothing changes

Reply:
Removed conditions from all the test and change the description